Connections aren't reset, they are closed. (#288)
quicwg/ops-drafts: draft-ietf-quic-applicability.md

QUIC does not provide an interface for exceptional handling of any stream.
If a stream that is critical for an application is closed, the application can
generate error messages on the application layer to inform the other end and/or

> the higher layer, which can eventually result in closure of the QUIC connection.

Reviewer: ```suggestion
the higher layer, which can eventually result in aborting of the QUIC connection.
```

> the higher layer, which can eventually result in closure of the QUIC connection.

Reviewer 2: ```suggestion
the higher layer, which can eventually terminate the QUIC connection.
```
